### DISCUSSION TITLE
Electron  apps not working (Obsidian)

### DISCUSSION THREAD

**Discussion by arghunter** [JaKooLit/Hyprland-Dots#479] [Q&A] [answered]
Electron based apps don't seem to worl. I know theres a way do fix it, but I can't find it. Can you help?

![image](https://github.com/user-attachments/assets/2fac4f23-d546-40af-957e-e9c129563b08)

**Comment by 0xl30** [comment]
Electron based apps Issue with  Adwaita cursor & icon 
Change Mouse Cursor/Icon Theme Issue Solved
1st -- Download Sweet Cursor Then Unzip And Paste In $HOME/.icons/
2nd Open GTK Settings Apps In App Menu Then Change Icon And Cursor

**Comment by arghunter** [comment]
I don't think thats the issue, I'm using breeze and flat (something) for cursor and icons

**Comment by 0xl30** [comment]
> I don't think thats the issue, I'm using breeze and flat (something) for cursor and icons

but it's true bro, you can fix change icon and cursor i'm facing same problemes around 1.5week ago
don't forget restart your pc after changing
![crash](https://github.com/user-attachments/assets/fb10fca8-6557-4d8f-871a-eac2048a1ba1)
https://www.reddit.com/r/archlinux/comments/1fjsgcx/segmentation_fault_core_dumped_in_a_lot_of_apps/

**Comment by Linxford** [comment]
i have Obsidian running well on my arch linux, without issues

check it out

i suggest you use the appImage

![image](https://github.com/user-attachments/assets/e2b46b2d-8418-468e-9447-d97ebe87bcfc)

**Comment by 0xl30** [comment]
> i have Obsidian running well on my arch linux, without issues
> 
> check it out
> 
> i suggest you use the appImage
> 
> ![image](https://private-user-images.githubusercontent.com/98231918/375423171-e2b46b2d-8418-468e-9447-d97ebe87bcfc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg1Nzk2NDMsIm5iZiI6MTcyODU3OTM0MywicGF0aCI6Ii85ODIzMTkxOC8zNzU0MjMxNzEtZTJiNDZiMmQtODQxOC00NjhlLTk0NDctZDk3ZWJlODdiY2ZjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDEwVDE2NTU0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA1ODFhZGNiNWZiMzJiYmRkMmRhYjMxYjkwYTcyY2MyOWU0NzBlNWY3NTBhYzZlODg0OGUwMjU3M2E1NGRkYTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.tsYF2MPS3pS8bXsMsIKu4VSSIEMRLpoZpg7c8sMwzvA)

I'm Facing segmentation fault in all Elections Apps with Adwaita Legacy Icon 
After changing icon issue fixed।
I think this issue related with Adwaita Legacy Icon Otherwise Wayland Flag।
I run obsidian in arch repo without any issue

**Comment by Linxford** [comment]
> > i have Obsidian running well on my arch linux, without issues
> > check it out
> > i suggest you use the appImage
> > ![image](https://private-user-images.githubusercontent.com/98231918/375423171-e2b46b2d-8418-468e-9447-d97ebe87bcfc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg1Nzk2NDMsIm5iZiI6MTcyODU3OTM0MywicGF0aCI6Ii85ODIzMTkxOC8zNzU0MjMxNzEtZTJiNDZiMmQtODQxOC00NjhlLTk0NDctZDk3ZWJlODdiY2ZjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDEwVDE2NTU0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA1ODFhZGNiNWZiMzJiYmRkMmRhYjMxYjkwYTcyY2MyOWU0NzBlNWY3NTBhYzZlODg0OGUwMjU3M2E1NGRkYTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.tsYF2MPS3pS8bXsMsIKu4VSSIEMRLpoZpg7c8sMwzvA)
> 
> I'm Facing segmentation fault in all Elections Apps with Adwaita Legacy Icon After changing icon issue fixed। I think this issue related with Adwaita Legacy Icon Otherwise Wayland Flag। I run obsidian in arch repo without any issue

good to hear bro.........

**Comment by JaKooLit** [comment]
Im moving this to discussion

as the issue is just another electron based

**Comment by brockar** [comment, marked as answer]
I installed Obsidian using yay (Arch) and am running it under xWayland, where it works very well.
To achieve this, I edited `/usr/share/applications/obsidian.desktop` and modified the `Exec=` line to:
```bash
Exec=/usr/bin/obsidian --ozone-platform=x11 --ozone-platform-hint=auto %U
```

**Reply by HarshalGunjalOp** [reply]
This worked for me. Thank you very much.
